### ISSUE TITLE
Doesn't build in C++11

### ISSUE THREAD

**Issue by esdrasdemorais** [sonosaurus/sonobus#146]
I was the same error below:

https://stackoverflow.com/questions/41615077/my-stdexchange-is-not-in-namespace-std

**Comment by essej** [comment]
No, it doesn't, it requires C++17 to build. Perhaps I need to make sure CMake doesn't let the configuration succeed without that, but I thought that it would.
